environments/go/builder/build.sh

```diff
@@ -1,19 +1,17 @@
 #!/bin/sh
 
+srcDir=${GOPATH}/src/$(basename ${SRC_PKG})
+trap "rm -rf ${srcDir}" EXIT
+
 if [ -d ${SRC_PKG} ]
 then
-    echo "Building in directory ${SRC_PKG}"
-    cd ${SRC_PKG}
-    go build -buildmode=plugin -i -o ${DEPLOY_PKG} .
+    echo "Building in directory ${srcDir}"
+    cp -r ${SRC_PKG} ${srcDir}
```

Comment:
Would symlink be faster instead of copying? Or does Go dislike symlinks for some reason?